docs/local_install_skaffold.md

```diff
@@ -108,81 +341,329 @@ Running the following command will do it for you (you'll be asked for your passw
-[backend-org-1-substra-backend-server-7bd48859c-dr9xj substra-backend] [15/Nov/2019 08:09:36] "GET /readiness HTTP/1.1" 200 2
-[backend-org-1-substra-backend-server-7bd48859c-dr9xj substra-backend] [15/Nov/2019 08:09:29] "GET /liveness HTTP/1.1" 200 2
-[backend-org-1-substra-backend-server-7bd48859c-dr9xj substra-backend] [15/Nov/2019 08:09:36] "GET /readiness HTTP/1.1" 200 2
+
+Launch Yarn in another terminal window:
+
+```sh
+yarn install
+
+API_URL=http://substra-backend.node-2.com yarn start
 ```
 
-- the `frontend` terminal displays:
+You will then have to map the frontend urls to your localhost, like this:
 
+```sh
+127.0.0.1 substra-frontend.node-1.com substra-frontend.node-2.com
 ```
-[substra-frontend-peer-2-substra-frontend-848764cb4c-j7qjc substra-frontend] REQUESTED ORIGINAL PATH: /
-[substra-frontend-peer-2-substra-frontend-848764cb4c-j7qjc substra-frontend] REQUESTED PARSED PATH: /
-[substra-frontend-peer-2-substra-frontend-848764cb4c-j7qjc substra-frontend] UNCACHABLE ROUTE /
-[substra-frontend-peer-2-substra-frontend-848764cb4c-j7qjc substra-frontend] REQUESTED ORIGINAL PATH: /login
-[substra-frontend-peer-2-substra-frontend-848764cb4c-j7qjc substra-frontend] REQUESTED PARSED PATH: /login
-[substra-frontend-peer-2-substra-frontend-848764cb4c-j7qjc substra-frontend] UNCACHABLE ROUTE /login
-[substra-frontend-peer-2-substra-frontend-848764cb4c-j7qjc substra-frontend] CHUNK NAMES [ 'user/components/index' ]
-[substra-frontend-peer-2-substra-frontend-848764cb4c-j7qjc substra-frontend] CACHING:  /login
+
+You can now head to <http://substra-frontend.node-2.com:3000/> and start to play with Substra!
+
+### Stop Substra
+
+In order to stop Substra, hit `ctrl + c` in each repository. On Ubuntu, if you want to stop the minikube Kubernetes cluster, you can use `minikube stop`.
+
+If you want to restart, you will just need to run again the `skaffold run` command in the 3 repositories.
+
+### Reset Substra
+
+You can reset your installation (if you've used `skaffold run`) with:
+
+```sh
+# run from each repositories (hlf-k8s, substra-backend, substra-frontend)
+skaffold delete
+# or
+kubectl rm ns peer-1 peer-2 orderer
+# On Ubuntu, to remove all the Kubernetes cluster
+minikube delete
 ```
 
 ## Login, password and urls
 
+### Credentials and urls
+
 Once the platform is running, you can sign in to the two organizations using the default development credentials:
 
 org-1:
+
 - API url: `http://substra-backend.node-1.com`
 - Frontend url: `http://substra-frontend.node-1.com`
 - Username: `node-1`
 - Password: `p@$swr0d44`
 
 org-2:
+
 - API url: `http://substra-backend.node-2.com`
 - Frontend url: `http://substra-frontend.node-2.com`
 - Username: `node-2`
 - Password: `p@$swr0d45`
+
+You should find the credentials in the charts: `skaffold.yaml` files or in the `substra-backend/charts/substra-backend/values.yaml`.
+
+### Browser extension
```

Comment:
It's really minor but I don't think this section should be part of the local intall with skaffold (and even the global documentation).

Using the backend webpage in the browser is in my opinion required only for development purposes and not for end users.

This section could leave only in the substra-backend README?

Reply:
Okay, noted, for now, I am moving this part to the troubleshooting section as well